src/content/blog/whats-new-december-2023.mdx

```diff
@@ -0,0 +1,134 @@
+
+- [Astro 4.0](https://github.com/withastro/astro/releases/tag/astro%404.0.0)
+- [Starlight 0.15](https://github.com/withastro/starlight/releases/tag/%40astrojs%2Fstarlight%400.15.0)
+
+## Docs
+
+@TODO:
+
+## Community
+
+### Community Awards November
+
+- 🌟 $250 awarded to 
+- 🥳 $250 awarded to 
+- 🎉 $250 awarded to 
+- 🍾 $250 awarded to 
+- 🎊 $250 awarded to 
+- 🎁 $250 awarded to 
+
+## Showcase
+
+### Content
+
+- [Nuxt 2 to Astro 3 Replatforming – from Setup to Production](https://stevenwoodson.com/blog/replatforming-from-nuxtjs-2-to-astro/) by @stevenwoodson
+- [Quick-and-useful: A DIY, self-hosted Linktree and Linkinbio clone with Astro and TinaCMS](https://darko.io/posts/linktree-and-linkinbio-clone-with-astro-and-tinacms/) by @_seiryuu_
+- [You are using Astro wrong](https://arshx.substack.com/p/you-are-using-astro-wrong-a-survival) by @arshxcx
+- [Launching a blog with Astro](https://goulven-clech.dev/2023/launching-blog-astro) by @goulvenclech
+- [Using RESEND (email API) with Astro](https://www.youtube.com/watch?v=HyDwVN1AFwY&t=2s) by @chrispennington
+- [A year working with HTML Web Components](https://hawkticehurst.com/writing/a-year-working-with-html-web-components/)
+- Community Call Deluxe from last week is on [YouTube](https://www.youtube.com/watch?v=VqfzAkI4AkY)
+- [https://jackcarey.co.uk/posts/astro-rewrite/](https://jackcarey.co.uk/posts/astro-rewrite/) by @jack_jack_jack
+- [Deep dive on View Transitions blog post series](https://fryuni.dev/deep-dive/astro-view-transitions/) by @Fryuni
+- [Astro Crash Course Video](https://www.youtube.com/watch?v=XoIHKO6AkoM)  by Brad Traversy
+- [Adding feature flags in Astro with Xata (showcasing View Transitions Forms) blog post](https://xata.io/blog/feature-flags-tutorial) by @rishirajjain
+
+### Themes
+
+- [Docbox](https://docbox-astro.vercel.app/) by @tffarhad
+- [ltree theme for Astro](https://github.com/DBozhinovski/astro-theme-ltree)
+- [Brutal](https://brutal.elian.codes) has reached over 150 stars by @eliancodes
+
+### Tools & tips
+
+With the release of the Astro Dev Toolbar, we also have Dev Toolbar apps to showcase now!
+
+- [Astro Dev Toolbar app for cloudflare](https://github.com/alexanderniebuhr/astro-cf-dev-toolbar-app-demo)
+- [Astro Tunnel app for Dev Toolbar](https://www.npmjs.com/package/astro-tunnel)
+- [Lighthouse app for Dev Toolbar](https://github.com/buk0vec/astro-lighthouse/tree/main)
+- [Cloudflare Dev Toolbar App (demo/WIP)](https://github.com/alexanderniebuhr/astro-cf-dev-toolbar-app-demo) by @alex
+- [Astro Font Picker devtoolbar app](https://github.com/randombits-dev/astro-font-picker) by @randombits.dev
+- [Astro Tunnel devtoolbar app](https://www.npmjs.com/package/astro-tunnel) by @shf0811
+- [astro-font](https://github.com/rishi-raj-jain/astro-font) by @rishirajjain
+- [Clerk + Astro demo](https://github.com/vin-e/clerk-astro-demo) by @vinny_code
+- [astro-heroicons](https://github.com/SeanMcP/astro-heroicons) updated by @seanmcp
+
+### Websites
```

Comment:
```suggestion
### Websites

Websites submitted to our Discord `#showcase` channel and featured on community calls this month.
```
I know we haven't done this before, but I think we should have a short sentence describing these categories going forward. These lists are getting quite long without anything to "read" around them!  :smile: